app/models/miq_filter.rb

```diff
@@ -58,7 +58,7 @@ def self.belongsto2object_list(tag)
         obj.children.grep(tag_part_klass).detect { |c| c.name == name }
       end
 
-      return result unless obj
+      return [] unless obj
```

Comment:
Not sure about this. tired of returning `[]` all over the place. Does it make sense to return `none` instead?

```suggestion
      return none unless obj
```

Reply:
`none` is something in ruby ?

`NameError (undefined local variable or method `none' for main:Object)`

@kbrock anyway we need `[]` because it means - that no object will displayed and other code count with `[]` in filterer  - but I agree `[]` is confusing -  so we need to bigger refactoring here - and I would leave it on other PR.

Reply:
Thanks. leave it as is. `none` is an Active Record thing. e.g.: `Vm.none`